internal/cli/actions.go

```diff
@@ -471,11 +487,12 @@ func openActionCmd(cli *cli) *cobra.Command {
 	}
 
 	cmd := &cobra.Command{
-		Use:     "open",
-		Args:    cobra.MaximumNArgs(1),
-		Short:   "Open action details page in the Auth0 Dashboard",
-		Long:    "Open action details page in the Auth0 Dashboard.",
-		Example: "auth0 actions open <id>",
+		Use:   "open",
+		Args:  cobra.MaximumNArgs(1),
+		Short: "Open the details page of an action",
+		Long:  "Open an action's details page in the Auth0 Dashboard.",
```

Comment:
**Reminder:** Update usage of "details page" to "settings page" for consistency.